tomcat/src/test/java/io/javaoperatorsdk/operator/sample/IntegrationTest.java

```diff
@@ -0,0 +1,122 @@
+package io.javaoperatorsdk.operator.sample;
+
+import io.fabric8.kubernetes.api.model.*;
+import io.fabric8.kubernetes.client.Config;
+import io.fabric8.kubernetes.client.ConfigBuilder;
+import io.fabric8.kubernetes.client.*;
+import io.fabric8.kubernetes.client.extended.run.RunConfigBuilder;
+import io.javaoperatorsdk.operator.Operator;
+import io.javaoperatorsdk.operator.config.runtime.DefaultConfigurationService;
+import org.junit.AfterClass;
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+import static java.util.concurrent.TimeUnit.*;
+import static org.awaitility.Awaitility.await;
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.notNullValue;
+
+public class IntegrationTest {
```

Comment:
Could we brake this method into smaller chuns based on semantics, it's kinda hard to follow this way.